### PR TITLE
Update index

### PR DESCRIPTION
**Pull request by KevinDenys**
More days options and added links to the posts you are deleting

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,22 +26,23 @@
...
             } else {
-                addPostListing('save', post.id, post.updated_time, message);
+                addPostListing('save', post.id, post.updated_time, message, groupid);
             }
         }
-        function addPostListing(state, id, time, text) {
-            var row = "<tr><td>" + id + "</td><td>" + time + "</td><td>" + text + "</td></tr>";
+        function addPostListing(state, id, time, text, groupid) {
+		var idsp = id.split('_')[1];
+            var row = "<tr><td>" + "<a href=https://www.facebook.com/groups/"+ groupid +"/permalink/" + idsp + ">" + idsp + "</td><td>" + time + "</td><td>" + text + "</td></tr>";
```
Comment (verticelo) [inline index.html:45]: I don't see the closing "< / a >" tag?

Comment (KevinDenys) [reply index.html:45]: <a> doesn't need a closing tag

**Comment by KevinDenys** [comment]
I changed the whole code, bootstrapped it, fixed the 0 days bug, and added checkboxes to delete only the things you want. Want me to upload it?

**Comment by KevinDenys** [comment]
I clicked on create new file but now it forked? I'm new to github so i have no idea if you can see the updated bootstrapped version? https://github.com/KevinDenys/fb-groups-delete-posts-older-than-x-days/blob/patch-1/indexbootstrapcheckboxes.html

File: index.html (hunk 1 above)
Comment (KevinDenys) [reply index.html:45]: But if you want to add one go ahead. In this case it doesn't matter

**Comment by verticelo** [comment]
I can see it yes. However, it seems that your default is to not delete posts and you would have to check every post to delete?

The most common use case for people using this script is to delete all posts older than X days, not wanting to check each and every post. Hence, only with that as default I could consider merging your change.

**Comment by KevinDenys** [comment]
Added a selectAll

**Pull request closed without merging**